src/cache.ts

```diff
@@ -58,4 +58,4 @@ export async function getLocalCDNFile(name: string, encoding?: BufferEncoding):
 
 export async function setLocalCDNFile(name: string, value: Buffer): Promise<void> {
 	await setCachedFile(`local_cdn:${name}`, value);
-}
\ No newline at end of file
+}
```

Comment:
Only putting this here once, but why did you change the ling endings of so many files?

Reply:
Said this already in the discord, but I thought we had discussed that this was how we wanted to be going forward? I realize this was probably not the right place for that, but I updated my linter and felt like it was weird to only apply it to the couple files I was working on for this pr

Reply:
Maybe re can revert this commit [`68bcd1f` (#91)](https://github.com/PretendoNetwork/account/pull/91/commits/68bcd1f8a6559d85ddf267660e15e022d4d870a5) and apply it on a different PR?

Reply:
That works for me, I'll go ahead and revert it